notebooks/run_model_in_notebook.ipynb

```diff
@@ -15,11 +15,9 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    path_to_config_file = path_to_model / "config_file_snellius.txt"
</pre><p>let's use absolute path to template like <code>path_to_config_file = "config_file_template.txt"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>